conans/test/integration/package_id/test_validate.py

```diff
@@ -262,3 +262,18 @@ def validate(self):
         c.save({"conanfile.py": conanfile})
         c.run("export-pkg . test/1.0@", assert_error=True)
         assert "Invalid ID: never ever" in c.out
+
+    def test_validate_install(self):
+        # https://github.com/conan-io/conan/issues/10602
+        c = TestClient()
+        conanfile = textwrap.dedent("""
+            from conan import ConanFile
+            from conan.errors import ConanInvalidConfiguration
+
+            class TestConan(ConanFile):
+                def validate(self):
+                    raise ConanInvalidConfiguration("never ever")
+            """)
+        c.save({"conanfile.py": conanfile})
+        c.run("install .")
+        assert "conanfile.py: ERROR: Invalid ID: never ever" in c.out
```

Comment:
Check the `Trying to install dependencies, but this configuration will fail to build a package` error?